.github/workflows/test.yml

```diff
@@ -23,18 +23,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: ./
-        name: 'Publish playmode results'
+        name: 'Publish test results'
         if: ${{ always() }}
         with:
-          checkName: 'Test Results (playmode)'
-          workdirPrefix: '/github/workspace/'
           githubToken: ${{ secrets.GITHUB_TOKEN }}
-          report: ./reports/playmode-results.xml
-      - uses: ./
-        name: 'Publish editmode results'
-        if: ${{ always() }}
-        with:
-          checkName: 'Test Results (editmode)'
-          workdirPrefix: '/github/workspace/'
-          githubToken: ${{ secrets.GITHUB_TOKEN }}
-          report: ./reports/editmode-results.xml
+          editModeReport: ./reports/editmode-results.xml
```

Comment:
Maybe it's not a big additional effort to use a pattern (or maybe multiple values with a comma separator) instead of two new inputs? I'm just concerned that changing API twice (from `report` to `editModeReport` and after back to `report` but allowing patterns) is not a good idea.